federation/cmd/federation-apiserver/app/server_test.go

```diff
@@ -281,12 +281,18 @@ func testCoreResourceList(t *testing.T) {
 	assert.Equal(t, "", apiResourceList.APIVersion)
 	assert.Equal(t, v1.SchemeGroupVersion.String(), apiResourceList.GroupVersion)
 
+	// Verify services.
 	found := findResource(apiResourceList.APIResources, "services")
 	assert.NotNil(t, found)
 	assert.True(t, found.Namespaced)
 	found = findResource(apiResourceList.APIResources, "services/status")
 	assert.NotNil(t, found)
 	assert.True(t, found.Namespaced)
+
+	// Verify events.
+	found = findResource(apiResourceList.APIResources, "events")
+	assert.NotNil(t, found)
+	assert.True(t, found.Namespaced)
 }
```

Comment:
might be worth also verifying that you don't find unexpected things? So that you don't accidentally turn on anything you don't want.